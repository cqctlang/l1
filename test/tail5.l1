foo = @lambda(x,y,z){
	return print(z);
};
foo(1,2,3);
foo = @lambda(x,y,z){
	return print(y);
};
foo(1,2,3);
foo = @lambda(x,y,z){
	return print(x);
};
foo(1,2,3);
bar = @lambda(x){
	return foo(x,x,x);
};
return bar(4);

