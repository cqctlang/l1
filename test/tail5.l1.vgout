entry: 0 params, 0 locals, 0 temps
L3: 3 params, 0 locals, 0 temps
L16: 3 params, 0 locals, 0 temps
L29: 3 params, 0 locals, 0 temps
L42: 1 params, 0 locals, 0 temps
<cval 3>
<cval 2>
<cval 1>
<cval 4>
