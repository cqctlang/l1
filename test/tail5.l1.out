entry: var params, 1 locals, 0 temps
L5: 3 params, 0 locals, 0 temps
L11: 3 params, 0 locals, 0 temps
L17: 3 params, 0 locals, 0 temps
L23: 1 params, 0 locals, 0 temps
<cval 3>
<cval 2>
<cval 1>
<cval 4>
